cmake/modules/patch-dpdk-conf.sh

```diff
@@ -50,6 +50,3 @@ setconf CONFIG_RTE_TEST_PMD n
 
 # async/dpdk does not like it
 setconf CONFIG_RTE_MBUF_REFCNT_ATOMIC n
-
-# balanced allocation of hugepages
-setconf CONFIG_RTE_EAL_NUMA_AWARE_HUGEPAGES n
```

Comment:
this breaks the build if DPDK enables `CONFIG_RTE_EAL_NUMA_AWARE_HUGEPAGES` by default, while libnuma-dev/libnuma-devel is not installed.